spearheadblocks/style.scss

```diff
@@ -0,0 +1,34 @@
+/*
+Theme Name: Spearhead Blocks
```

Comment:
You can remove this .scss file.  It doesn't look like there will be any CSS compilation for this theme so I don't think it will be needed at all.